### PR TITLE
use oc set env in extended tests

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

File: test/extended/images/hardprune.go [hunk 1]
```diff
@@ -461,7 +461,7 @@ func ConfigureRegistry(oc *exutil.CLI, desiredState RegistryConfiguration) (bool
 		return false, nil
 	}
 
-	if err := oc.Run("env").Args(append([]string{"dc/docker-registry"}, envOverrides...)...).Execute(); err != nil {
+	if err := oc.Run("set").Args(append([]string{"set", "dc/docker-registry"}, envOverrides...)...).Execute(); err != nil {
```
Comment (deads2k) [inline test/extended/images/hardprune.go]: typo

**Comment by deads2k** [comment]
/lgtm

I plan to green button on green image-registry to unstick other pulls.

**Comment by bparees** [comment]
> I plan to green button on green image-registry to unstick other pulls.

should really wait for image_eco also since that was impacted.

**Comment by bparees** [comment]
image eco and registry have passed, pressing the button.

**Comment by deads2k** [comment]
> image eco and registry have passed, pressing the button.

pressed!

**Pull request merged**